docs/CONTRIBUTING.md

```diff
@@ -36,7 +36,8 @@ This guide provides instructions for contributing to Spark Android. It covers en
 
 ### Repository Access
 
-> [!NOTE] Leboncoin employees only
+> [!NOTE] 
+> Leboncoin employees only
```

Comment:
You'll need two trailing spaces for a newline here.

Reply:
one or none is enough it seems, Github render the alert now!